age.md

```diff
@@ -12,8 +12,10 @@ ABNF syntax follows [RFC 5234][] and [RFC 7405][] and references the core rules
 in RFC 5234, Appendix B.1.
 
 The base64 encoding used throughout is the standard Base 64 encoding specified
-in [RFC 4648][], Section 4. Encoders MUST generate canonical base64 according to
-RFC 4648, Section 3.5, and decoders MUST reject non-canonical encodings.
+in [RFC 4648][], Section 4, without `=` padding characters (sometimes referred
```

Comment:
Is this clear enough that the Base64 used in PEM is not subject to these requirements? The armor section references Base64 by name, so it's a potential point of confusion.